zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -671,8 +671,8 @@ function ParagraphCtrl ($scope, $rootScope, $route, $window, $routeParams, $loca
       })
 
       if (navigator.appVersion.indexOf('Mac') !== -1) {
-        $scope.editor.setKeyboardHandler('ace/keyboard/emacs')
```

Comment:
@felixcheung 

Zeppelin specifies the editor shortcuts.

- `CTRL-S` doesn't work even in master.
- `CTRL-Y` is for paste. But every os has their own binding for paste. (e.g `CMD-V` in OSX, `CTRL-V` in Windows). So not sure we need to keep this one.

So except 2 specified shortcuts, all shortcuts works well.

![image](https://cloud.githubusercontent.com/assets/4968473/26705730/078d3b50-4773-11e7-8ea2-c10515a61fe7.png)
![image](https://cloud.githubusercontent.com/assets/4968473/26705734/0dd48360-4773-11e7-9902-05e6ce86bcbe.png)